src/sync/streaming/SSEHandler/index.ts

```diff
@@ -82,11 +82,16 @@ export default function SSEHandlerFactory(log: ILogger, pushEmitter: IPushEventE
             parsedData.changeNumber,
             parsedData.segmentName);
           break;
+        // @TODO remove
```

Comment:
Don't remove it. The notification type could still be valid in certain conditions that will be handled at the service level.